vmdb/app/models/container_definition.rb

```diff
@@ -2,4 +2,5 @@ class ContainerDefinition < ActiveRecord::Base
   # :name, :image, :image_pull_policy, :memory, :cpu
   belongs_to :container_group
   has_many :container_port_configs, :dependent => :destroy
+  has_one :container
```

Comment:
Please also add its counterpart in `class Container`:

    belongs_to :container_definition